dev/com.ibm.ws.microprofile.reactive.streams.operators_fat_tck/publish/tckRunner/tck/src/test/java/com/ibm/ws/microprofile/reactive/streams/operators/test/ArchiveProcessor.java

```diff
@@ -0,0 +1,46 @@
+/*******************************************************************************
+ * Copyright (c) 2018 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package com.ibm.ws.microprofile.reactive.streams.operators.test;
+
+import java.io.File;
+import org.jboss.arquillian.container.test.spi.client.deployment.ApplicationArchiveProcessor;
+import org.jboss.arquillian.test.spi.TestClass;
+import org.jboss.shrinkwrap.api.Archive;
+import org.jboss.shrinkwrap.api.spec.JavaArchive;
+import org.jboss.shrinkwrap.resolver.api.maven.Maven;
+
+
+import org.jboss.shrinkwrap.resolver.api.maven.PomEquippedResolveStage;
+import com.beust.jcommander.Parameters;
+
+/**
+ * We weave in the hamcrest jar that is used by some of the microprofile tck tests.
+ * The build.gradle file pull the hamcrest jar from maven and puts it in the lib directory
+ */
+public class ArchiveProcessor implements ApplicationArchiveProcessor {
+
+    /* (non-Javadoc)
+     * @see org.jboss.arquillian.container.test.spi.client.deployment.ApplicationArchiveProcessor#process(org.jboss.shrinkwrap.api.Archive, org.jboss.arquillian.test.spi.TestClass)
+     */
+    @Override
+    public void process(Archive<?> applicationArchive, TestClass testClass) {
+        System.out.println("WLP: Processing archive: " + applicationArchive.toString());
+        if (applicationArchive instanceof JavaArchive) {
+            //Needed by some tck classes. E.G. org.reactivestreams.tck.flow.support.HelperPublisher
+            ((JavaArchive) applicationArchive).addPackages(true, "org.reactivestreams.example");
+
+            //jcommander is needed by testng. E.G. org.testng.CommandLineArgs
+            //This really should be added as a library. However due to the structure of the tck this processer can only see a jar file and not the war it eventually gets wrapped into
+            JavaArchive jCommander = Maven.resolver().resolve("com.beust:jcommander:1.72").withTransitivity().asSingle(JavaArchive.class);          
```

Comment:
How does this relate to where hamcrest appears in build.gradle does jCommander need to be in build.gradle
and do you need to add it to files in cnf/oss_* or get approval or put it in artifactory?

Reply:
I don't think jCommander needs to be in build.gradle since it's coming directly from maven. 

jcommander is already in artifactory so I think the best thing to do is change the version to match what w ealready have.